source/content/mu-plugin.md

```diff
@@ -9,18 +9,26 @@ tags: [plugins]
 
 For actions or filters you want to run even when a theme's `functions.php` isn't invoked by a request, or before plugins are loaded by WordPress, you can create a [Must-Use (**MU**) Plugin](https://codex.wordpress.org/Must_Use_Plugins).
 
-MU-Plugins are activated by default by adding a PHP file to the `wp-content/mu-plugins` directory. It affects the whole site, including all sites under a WordPress Multisite installation.
+MU-plugins are activated by default by adding a PHP file to the `wp-content/mu-plugins` directory. It affects the whole site, including all sites under a WordPress Multisite installation.
 
 MU-plugins are loaded by PHP in alphabetical order, before normal plugins. This means API hooks added in an MU-plugin apply to all other plugins even if they run hooked-functions in the global namespace.
 
-## Why use MU Plugins?
+## Why Use MU Plugins?
+
 While you can add code in the `wp-config.php` file for site-wide behavior, actions and filters shouldn't be added here.
 
 If they are added above the `require_once ABSPATH . 'wp-settings.php';` statement, the WordPress site will get a Fatal PHP error because the `add_action()` and `add_filter()` functions won't be defined yet.
 
 If they are added below the `require_once ABSPATH . 'wp-settings.php';` statement, then the entirety of WordPress has already been loaded and the actions / filters won't be applied, or would be applied last.
 
+### Use wp_get_environment_type for Environment-specific Actions
+
+<Partial file="wp_get_environment_type.md" />
+
+An MU-plugin can be instructed to run or perform environment-specific actions. Use `wp_get_environment_type` to pass the environment to the plugin.
```

Comment:
> Use `wp_get_environment_type` to look up the current environment in a platform-neutral way.

Maybe link to the docs here.

Reply:
updated the copy to match your suggestion and use "look up the current" instead of "pass the"
I haven't  found any proper docs on the feature, so for now, the link in the partial (L26) just includes a link to [the wp blog](https://make.wordpress.org/core/2020/07/24/new-wp_get_environment_type-function-in-wordpress-5-5/)

Reply:
Looks like it has not made it into the WordPress codex, eventually should be found [in this list](https://codex.wordpress.org/Function_Reference).